tests/defences/test_strip.py

```diff
@@ -0,0 +1,108 @@
+# MIT License
```

Comment:
Could you please convert these tests to the new pytest pattern: https://github.com/Trusted-AI/adversarial-robustness-toolbox/wiki/ART-Unit-Testing

Reply:
Done